acceptance/suites/tests/authorization/hup_reloads_crl.rb

```diff
@@ -0,0 +1,68 @@
+test_name '(SERVER-1380) CRL should reload after a server HUP' do
```

Comment:
I added a `skip_test` statement at the top of the test to accommodate this. I did write the test to run and "pass" in the no-non-master-agent case, but perhaps skipping it entirely is the smarter approach.